apps/store/.env.local.example

```
@@ -37,11 +37,12 @@ SKIP_BUILD_STATIC_GENERATION=<true>
 
 # Feature Flags
 NEXT_PUBLIC_FEATURE_INSURELY=<true>
-NEXT_PUBLIC_FEATURE_ENGLISH_LANGUAGE=<true>
-NEXT_PUBLIC_FEATURE_COUNTRY_SELECTOR=<true>
-NEXT_PUBLIC_FEATURE_SUCCESS_ANIMATION=<true>
-NEXT_PUBLIC_FEATURE_PET_INSURANCE=<true>
-NEXT_PUBLIC_FEATURE_STOREFRONT_CONNECT_PAYMENT=<true>
+NEXT_PUBLIC_FEATURE_ENGLISH_LANGUAGE=true
+NEXT_PUBLIC_FEATURE_COUNTRY_SELECTOR=true
+NEXT_PUBLIC_FEATURE_SUCCESS_ANIMATION=true
+NEXT_PUBLIC_FEATURE_PET_INSURANCE=true
+NEXT_PUBLIC_FEATURE_STOREFRONT_CONNECT_PAYMENT=true
+NEXT_PUBLIC_FEATURE_SAS_PARTNERSHIP=true
```

Just curious:
Why not just to add it like the following? Or maybe change `NEXT_PUBLIC_FEATURE_INSURELY` as along with these changes.

```suggestion
NEXT_PUBLIC_FEATURE_ENGLISH_LANGUAGE=<true>
NEXT_PUBLIC_FEATURE_COUNTRY_SELECTOR=<true>
NEXT_PUBLIC_FEATURE_SUCCESS_ANIMATION=<true>
NEXT_PUBLIC_FEATURE_PET_INSURANCE=<true>
NEXT_PUBLIC_FEATURE_STOREFRONT_CONNECT_PAYMENT=<true>
NEXT_PUBLIC_FEATURE_SAS_PARTNERSHIP=<true>
```